api/Element.json

```diff
@@ -1964,43 +1964,43 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/Element/innerHTML",
           "support": {
             "webview_android": {
-              "version_added": null
+              "version_added": true
```

Comment:
Please sort this alphabetically (#1882/https://github.com/mdn/browser-compat-data/issues/398), and see issue #2690.

Reply:
Done

Reply:
Done, but TBH I don't get the point of taking care of this in this PR.

It's unrelated and add noise.
And doesn't it introduce more merge conflicts than it solves?

Please don't discuss about it here, to not introduce even more noise.